docs/_documentation/using_per_event_authorization.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Event-Based Authorization
+position: 13
+---
+
+## Event-Based Authorization using EventAuthField
+
+Nakadi provides an option to specify `event_auth_field` in an event type definition, which can be used to differentiate
+ consumption of events in an event type on a per-consumer level. The `event_auth_field` takes the path to a string field
+ and type of a field inside an event that can be then used by an authorization plugin to classify if a consumer
+ should receive an event published to the event type. It is optional for the field to be part of the event type's
```

Comment:
The fact that it's optional can be omitted in my opinion. This can be inferred from schema definition.